### ISSUE TITLE
AboutYou.aspx

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/HW8#10]
<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="AboutYou.aspx.cs" Inherits="Kotturu2.AboutYou" %>

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>    
    <form id="form1" runat="server">
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-header">
                <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="brand" href="#">
                    <img src="Logo.PNG" class="ui-icon-image" height="50" width="50" alt="" />
                </a>
            </div>

            <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
                <ul class="nav navbar-nav">
                    <li><a href="Home.aspx">Home</a></li>
                    <li><a href="Login.aspx">Login</a></li>
                    <li  class="active"><a href="AboutYou.aspx">About You</a></li>
                </ul>
            </div>
            </div>
        </div>

 <div class="container" style="margin-top: 3%">
            <fieldset>
                
                <div style="margin-left: 25%"><h2>Please tell us a little bit about you to get started</h2></div>
                <hr />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="f_name">First Name</label>
                    <div class="col-md-6">
                        <input id="f_name" name="f_name" type="text" placeholder="Enter Your First Name " class="form-control input-md" required="required"/>
                        <br />
                    </div>
                </div>
                <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="l_name">Last Name</label>
                    <div class="col-md-6">
                        <input id="l_income" name="l_income" class="form-control" placeholder="Enter Your Last Name" type="text" required="required"/>
                        <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="address">Address</label>
                    <div class="col-md-6">
                            <input id="address" name="address" type="text" placeholder="Enter Address" class="form-control input-md" required="required"/>
                            <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="city">City</label>
                    <div class="col-md-6">
                            <input id="city" name="city" class="form-control" placeholder="Enter City Name" type="text" required="required"/>
                                                    <br />

                        </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="state">State</label>
                    <div class="col-md-6">
                            <input id="state" name="state" class="form-control" placeholder="Enter State Name" type="text" required="required"/>
                            <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="zip">Enter Zipcode</label>
                    <div class="col-md-6">
                        <input id="zip" name="zip" class="form-control"  placeholder="#####" type="number" required="required"/>
                        <br />
                    </div>
                </div>
                    <br />

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="email">Enter Email</label>
                    <div class="col-md-6">
                        <input id="email" name="email" class="form-control" placeholder="username@me.com" type="email" required="required" />
                        <br />
                    </div>
                </div>
                    <br />

                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="bday">Enter Birthday</label>
                    <div class="col-md-6">
                        <input id="bday" name="bday" class="form-control" placeholder="MM-DD-YYYY" type="date" required="required"/>
                        <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="phoneNumber">Enter Phone Number</label>
                    <div class="col-md-6">
                        <input id="phoneNumber" name="phoneNumber" class="form-control" placeholder="###-###-####" type="tel" required="required"/> 
                        <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="zip">Enter Questions or Comments (Optional): </label>
                    <div class="col-md-6">
                        <input id="comments" name="comments" class="form-control" placeholder="Text" type="text"/>
                        <br />
                    </div>

                </div>
<br />
                    <br />

                <div class="form-group">
                     <div class="col-md-6">
                    <asp:Button runat="server" class="btn btn-md btn-success"  id="btnSubscribe" name="btnSubscribe" Text="Subscribe to the Mailing List" OnClick="btnSubscribe_Click1" ></asp:Button>
                         <br />
                    </div>
                </div>
                <br />

                <div class="form-group">
                    <div class="col-md-12">
                        <br />
                        <asp:Label id="lblSuccess" name="lblSuccess" runat="server" style="position: relative" Text="Success!  You have successfully subscribed to the mailing list!"></asp:Label>
                    </div>
                </div>

                <br />
           </fieldset>
    </form>
</div>
             </form>
            </body>
      

</html>

**Comment by pallavikotturu** [comment]
<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="AboutYou.aspx.cs" Inherits="Kotturu2.AboutYou" %>

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>  
    <form id="form1" runat="server">

 
                <div style="margin-left: 25%"><h2>Please tell us a little bit about you to get started</h2></div>
                <hr />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="f_name">First Name</label>
                    <div class="col-md-6">
                        <input id="f_name" name="f_name" type="text" placeholder="Enter Your First Name " class="form-control input-md" required="required"/>
                        <br />
                    </div>
                </div>
                <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="l_name">Last Name</label>
                    <div class="col-md-6">
                        <input id="l_income" name="l_income" class="form-control" placeholder="Enter Your Last Name" type="text" required="required"/>
                        <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="address">Address</label>
                    <div class="col-md-6">
                            <input id="address" name="address" type="text" placeholder="Enter Address" class="form-control input-md" required="required"/>
                            <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="city">City</label>
                    <div class="col-md-6">
                            <input id="city" name="city" class="form-control" placeholder="Enter City Name" type="text" required="required"/>
                                                    <br />

```
                    </div>
            </div>
                <br />
            <!-- Select Basic -->
            <div class="form-group">
                <label class="col-md-4 control-label" for="state">State</label>
                <div class="col-md-6">
                        <input id="state" name="state" class="form-control" placeholder="Enter State Name" type="text" required="required"/>
                        <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="zip">Enter Zipcode</label>
                <div class="col-md-6">
                    <input id="zip" name="zip" class="form-control"  placeholder="#####" type="number" required="required"/>
                    <br />
                </div>
            </div>
                <br />

            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="email">Enter Email</label>
                <div class="col-md-6">
                    <input id="email" name="email" class="form-control" placeholder="username@me.com" type="email" required="required" />
                    <br />
                </div>
            </div>
                <br />

            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="bday">Enter Birthday</label>
                <div class="col-md-6">
                    <input id="bday" name="bday" class="form-control" placeholder="MM-DD-YYYY" type="date" required="required"/>
                    <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="phoneNumber">Enter Phone Number</label>
                <div class="col-md-6">
                    <input id="phoneNumber" name="phoneNumber" class="form-control" placeholder="###-###-####" type="tel" required="required"/> 
                    <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="zip">Enter Questions or Comments (Optional): </label>
                <div class="col-md-6">
                    <input id="comments" name="comments" class="form-control" placeholder="Text" type="text"/>
                    <br />
                </div>

            </div>
```

<br />
                    <br />

```
            <div class="form-group">
                 <div class="col-md-6">
                <asp:Button runat="server" class="btn btn-md btn-success"  id="btnSubscribe" name="btnSubscribe" Text="Subscribe to the Mailing List" OnClick="btnSubscribe_Click1" ></asp:Button>
                     <br />
                </div>
            </div>
            <br />

            <div class="form-group">
                <div class="col-md-12">
                    <br />
                    <asp:Label id="lblSuccess" name="lblSuccess" runat="server" style="position: relative" Text="Success!  You have successfully subscribed to the mailing list!"></asp:Label>
                </div>
            </div>

            <br />

            <div class="col-page col-sm-12 col-md-12">
            <div class="img-publicacion-home">
                <p>
                    <br />
                    <br />
                    <hr />
                    Footer:  
<a href="https://www.linkedin.com/in/pallavikotturu"><span class="label label-info">Author</span></a> 
<a href="https://cis.temple.edu/"><span class="label label-info">Temple University CIS</span></a> 
| <i class="icon-calendar"></i> November 8th, 2016 at 10:15 am
| <i class="icon-comment"></i> <a href="#">3 Comments</a>
| <i class="icon-share"></i> <a href="#">39 Shares</a>
```

</p>

```
                </div>
              </div>
       </fieldset>
</form>
```

</div>
             </form>
            </body>

</html>

**Comment by pallavikotturu** [comment]
`<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="AboutYou.aspx.cs" Inherits="Kotturu2.AboutYou" %>



<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>  
    <form id="form1" runat="server">

```
    <!-- This is the navbar - the required standard website navigation element that appears on every page.  It allows the user to navigate among various pages within the sites -->
<div class="navbar navbar-inverse navbar-fixed-top">
    <div class="container">
        <div class="navbar-header">
            <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="brand" href="#">
                <img src="Logo.PNG" class="ui-icon-image" height="50" width="50" alt="" />
            </a>
        </div>

        <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
            <ul class="nav navbar-nav">
                <li><a href="Home.aspx">Home</a></li>
                <li><a href="Login.aspx">Login</a></li>
                <li  class="active"><a href="AboutYou.aspx">About You</a></li>
            </ul>
        </div>
        </div>
    </div>


     <!-- This page is meant to essentially gather information about the visitor of the site who wishes to ‘subscribe’ to the motivational material email list.  The required validations are included.-->
```

 <div class="container" style="margin-top: 3%">
            <fieldset>
                
                <div style="margin-left: 25%"><h2>Please tell us a little bit about you to get started</h2></div>
                <hr />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="f_name">First Name</label>
                    <div class="col-md-6">
                        <input id="f_name" name="f_name" type="text" placeholder="Enter Your First Name " class="form-control input-md" required="required"/>
                        <br />
                    </div>
                </div>
                <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="l_name">Last Name</label>
                    <div class="col-md-6">
                        <input id="l_income" name="l_income" class="form-control" placeholder="Enter Your Last Name" type="text" required="required"/>
                        <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="address">Address</label>
                    <div class="col-md-6">
                            <input id="address" name="address" type="text" placeholder="Enter Address" class="form-control input-md" required="required"/>
                            <br />
                    </div>
                </div>
                    <br />
                
                <div class="form-group">
                    <label class="col-md-4 control-label" for="city">City</label>
                    <div class="col-md-6">
                            <input id="city" name="city" class="form-control" placeholder="Enter City Name" type="text" required="required"/>
                                                    <br />

```
                    </div>
            </div>
                <br />
            <!-- Select Basic -->
            <div class="form-group">
                <label class="col-md-4 control-label" for="state">State</label>
                <div class="col-md-6">
                        <input id="state" name="state" class="form-control" placeholder="Enter State Name" type="text" required="required"/>
                        <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="zip">Enter Zipcode</label>
                <div class="col-md-6">
                    <input id="zip" name="zip" class="form-control"  placeholder="#####" type="number" required="required"/>
                    <br />
                </div>
            </div>
                <br />

            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="email">Enter Email</label>
                <div class="col-md-6">
                    <input id="email" name="email" class="form-control" placeholder="username@me.com" type="email" required="required" />
                    <br />
                </div>
            </div>
                <br />

            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="bday">Enter Birthday</label>
                <div class="col-md-6">
                    <input id="bday" name="bday" class="form-control" placeholder="MM-DD-YYYY" type="date" required="required"/>
                    <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="phoneNumber">Enter Phone Number</label>
                <div class="col-md-6">
                    <input id="phoneNumber" name="phoneNumber" class="form-control" placeholder="###-###-####" type="tel" required="required"/> 
                    <br />
                </div>
            </div>
                <br />
            <!-- Text input-->
            <div class="form-group">
                <label class="col-md-4 control-label" for="zip">Enter Questions or Comments (Optional): </label>
                <div class="col-md-6">
                    <input id="comments" name="comments" class="form-control" placeholder="Text" type="text"/>
                    <br />
                </div>

            </div>
```

<br />
                    <br />

```
            <div class="form-group">
                 <div class="col-md-6">
                <asp:Button runat="server" class="btn btn-md btn-success"  id="btnSubscribe" name="btnSubscribe" Text="Subscribe to the Mailing List" OnClick="btnSubscribe_Click1" ></asp:Button>
                     <br />
                </div>
            </div>
            <br />

            <div class="form-group">
                <div class="col-md-12">
                    <br />
                    <asp:Label id="lblSuccess" name="lblSuccess" runat="server" style="position: relative" Text="Success!  You have successfully subscribed to the mailing list!"></asp:Label>
                </div>
            </div>

            <br />


            <!-- required footer with links to my linkedin page and the cis temple page-->
            <div class="col-page col-sm-12 col-md-12">
            <div class="img-publicacion-home">
                <p>
                    <br />
                    <br />
                    <hr />
                    Footer:  
<a href="https://www.linkedin.com/in/pallavikotturu"><span class="label label-info">Author</span></a> 
<a href="https://cis.temple.edu/"><span class="label label-info">Temple University CIS</span></a> 
| <i class="icon-calendar"></i> November 8th, 2016 at 10:15 am
| <i class="icon-comment"></i> <a href="#">3 Comments</a>
| <i class="icon-share"></i> <a href="#">39 Shares</a>
```

</p>

```
                </div>
              </div>
       </fieldset>
</form>
```

</div>
             </form>
            </body>

</html>
`
